src/Integration/RequestIntegration.php

```diff
@@ -144,13 +144,16 @@ private function removePiiFromHeaders(array $headers): array
     {
         $keysToRemove = ['authorization', 'cookie', 'set-cookie', 'remote_addr'];
```

Comment:
Sure thing.  While we're at it are we suuuuure we don't want to just move it into Options?  Seems like it would be great to have that flexibility, but just tell me what you'd like @ste93cry .

Reply:
I don't have a strong contrary opinion about having a configurable set of options, although I would prefer to implement them as options of the integration itself rather than of the SDK, but since I also believe that Relay is the right tool for this job and knowing the we said these kind of things should not be in the SDKs, I will let you decide how to proceed @HazAT

Reply:
Yeah I can understand that.  Happy to change it back if you'd prefer just let me know.

Reply:
So, I've talked on Discord with @HazAT and we agreed to expose this option as option of the integration itself. You can take a look at the `IgnoreErrorsIntegration` class to see how we implemented something similar 😃 I will review again the code once you submit all the new changes

Reply:
@ste93cry hey, 

> @soundsgoodsofar do you still have plans to finish the work? Otherwise, if it's ok for you I would like to do it for you so that we can merge the PR

Hey @ste93cry , thanks for checking.  I've gotten a bit busy on other things and I'm not sure I'll be able to get back to this immediately.  If you're able to help implement that would be much appreciated.